.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,19 +1,33 @@
-## Testing
+## ℹ️ Description
 
 <!--
-What is needed to test the changes? e.g. specific cluster, service definitions
-How can one see the result of your work? e.g. configurations, URLs
+What is the change your PR does?
+Please add a short description here. (one sentence is enough)
 -->
 
-## Trade-offs
+- 🔖 JIRA: [DCOS-12345](https://jira.mesosphere.com/browse/DCOS-12345)
```

Comment:
Why not keep the jira in the title?

Reply:
I'm not saying that we should remove it there… This _Link_ helps speeding up reviewing. Currently we have to copy paste (or write scripts like you did)

Reply:
Maybe I should polish my script and share it. I'll try to do if I find time today 👍

Reply:
No. Your script isnt solving the issue, but only working around it. You can use it, you can give it to the team, but what about OSS issues and OSS community users?  

I'm totally aware that this adds 2-5 seconds to creating a PR, and I agree that there _should_ be a better process (use github issues), but we as a company decided to use JIRA - now we find the best solution for _everyone_

Reply:
Will do once this is merged. I honestly don't like this, because it contradicts the goal of making this template as easy to fill out as possible while also providing every information the reviewer needs. But if the team wants it that is also fine for me, I won't block it because of this, but I will also not approve.

Reply:
I like the Idea having a template that is as easy to use as possible. 
Also I like the idea having a template that provides all the information the reviewer needs.

I do not understnd your issue tho - especially because  _you_ are the person who _wrote a f*ing script_ for these links… are these 2-5 seconds when creating a PR _that bad_?

Reply:
Language please!

Reply:
They are bad to me because it's another hoop to jump through and because they are benefitial the situation where you are coming from the github UI instead of jira for reviews (like I did when I was pinged here, guilty 🙈). I would like us to go the other way, because we as a team decided that Jira is our single source of truth regarding issue states

Reply:
have you ever tried to walk through the JIRA board? 
1. be on the board
2. open card
3. scroll down
4. open layer
5. click pr

…and if you tried to to one of these steps in a new tab, you have to start over again…